### DISCUSSION TITLE
shiny.samples.Sample.Maui.BlePerf isn't working.

### DISCUSSION THREAD

**Discussion by sotakoyama** [shinyorg/shiny#1173] [Q&A] [answered]
I want to transfer data using BLE.
As a preliminary step, I tried to connect the device on Shiny's Sample.

However, [Sample ](https://github.com/shinyorg/shiny/tree/master/samples/Sample.Maui/BlePerf) doesn't seem to work.

I pressed Toggle GATT Connection on 'BLE - Pref Client' and Toggle GATT Server on 'BLE - Pref Server', but could not connect.

![Screenshot_20230110-150221](https://user-images.githubusercontent.com/91939052/211477852-dd8c0781-be29-48ad-88a2-8918eac05905.png)
![Screenshot (2023_01_10 15_02_26)](https://user-images.githubusercontent.com/91939052/211477860-86283e9f-1978-4e6f-a7f9-00ebd492c804.png)

I know that the current version is alpha.
However, looking at other people's discussions, it appears to work fine.

Is this a bug that happened because Sample is still under development?
Or am I doing something wrong?

Any advice would be appreciated.
Thank you very much.

**Comment by aritchie** [comment, marked as answer]
This is why Shiny v3 is marked alpha.  It isn't done.  Samples were ported to keep people from bugging me.  Some "samples" are for my testing.

**Reply by sotakoyama** [reply]
Thank you for your answer!
